cpp/src/rolling/detail/range_utils.cuh

```diff
@@ -413,209 +413,226 @@ template <typename T, typename V>
   }
 }
 
+template <direction Direction, typename OrderbyT, typename Comparator>
+__device__ inline auto compute_bounded(column_device_view::const_iterator<OrderbyT> const& begin,
```

Comment:
I removed moved this out of the body of the bounded function I don't think it helps much. Inlining it as a lambda seems like a good compromise to avoid duplication.